presto-main/src/test/java/com/facebook/presto/sql/planner/iterative/rule/TestPruneCountAggregationOverScalar.java

```diff
@@ -41,10 +43,11 @@
 public class TestPruneCountAggregationOverScalar
         extends BaseRuleTest
 {
+    private static final FunctionManager TEST_FUNCTION_MANAGER = createTestMetadataManager().getFunctionManager();
```

Comment:
Rather than creating a new function manager, add a helper in `RuleTester`
```java
public FunctionManager getFunctionManager()
{
    return queryRunner.getMetadata().getFunctionManager();
}